core/processors/__init__.py

```
@@ -1,26 +1,45 @@
-    "ProcessingResult",
     "ChunkingStrategy",
     "TokenBasedChunking",
     "SemanticChunking",
     "SlidingWindowChunking",
-    "ChunkingStrategyFactory"
+    "ChunkingStrategyFactory",
+    # Deprecated - kept for backward compatibility
+    "DocumentProcessor",
+    "ProcessingConfig",
+    "ProcessingResult",
 ]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)